hack/generate-deployment-dev

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+# Copyright 2019 Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setup
+PROD_YAML='manifests/deployment.yml'
+DEV_YAML='manifests/deployment-dev.yml'
+PROD_IMAGE="$1"
+DEV_IMAGE="$2"
+
+# Check
+( [ -z "$PROD_IMAGE" ] || [ -z "$DEV_IMAGE" ] ) && exit 5
```

Comment:
Why the subshell?. Also, I think it would be much better if we print a helpful error message the helps the user to understand how the script is used. Same below.

Reply:
Thx for your review. You're right. The subshell is not required. I've fixed this and helpful error messages were added.